src/transformers/pipelines/depth_estimation.py

```diff
@@ -50,12 +51,12 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, "vision")
         self.check_model_type(MODEL_FOR_DEPTH_ESTIMATION_MAPPING_NAMES)
 
-    def __call__(self, images: Union[str, List[str], "Image.Image", List["Image.Image"]], **kwargs):
+    def __call__(self, inputs: Union[str, List[str], "Image.Image", List["Image.Image"]] = None, **kwargs):
```

Comment:
Note for myself: this change won't break Inference API since it's using positional argument and not keyword one (see [here](https://github.com/huggingface-internal/api-inference/blob/ef27d885a6a9c0228fa1efd4b4b730bfb007e012/shard/app/main.py#L188) - private link). So all good to change it, same for all other pipelines :)